HW11/homework_11.py

```diff
@@ -0,0 +1,49 @@
+import time
+from datetime import datetime
+
+
+class ContextManager:
+    def __enter__(self):
+        print("="*10)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print("="*10)
+
+
+with ContextManager():
+    try:
+        def name_time_decorator(func):
+            def inner1(*args, **kwargs):
+                print(f"Function {func.__name__} started at {datetime.now()}")
+                start_time = time.time()
+                func(*args, **kwargs)
```

Comment:
write the result of the func call to some variable and return in after the last print to make it work correctly with functions that return some value